nabla/test/curve.ts

```diff
@@ -1,217 +0,0 @@
-/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
I removed this test file as I used it only to analyze and understand the old curve contract. It has no relevance anymore.